modules/nf-core/seqkit/rmdup/tests/main.nf.test

```diff
@@ -0,0 +1,173 @@
+nextflow_process {
+
+    name "Test Process SEQKIT_RMDUP"
+    script "../main.nf"
+    process "SEQKIT_RMDUP"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "seqkit"
+    tag "seqkit/rmdup"
+
+    test("sarscov2-genome_fasta") {
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
```

Comment:
Can you update the fetching of the test data to the new [guidelines](https://nf-co.re/docs/guidelines/components/modules#input-data)